frontend/src/reducks/users/operations.js

```diff
@@ -0,0 +1,55 @@
+import API from "../../API";
+import { fetchUsersAction, signInAction, signUpAction } from "./actions";
+
+const api = new API();
+const LOGIN_USER_KEY = "LOGIN_USER_KEY";
```

Comment:
Please change "LOGIN_USER_KEY" to "E_LIKE_LOGIN_USER_KEY".
I used the same key in the other project, and that causes the bug.